_maps.yml

```diff
@@ -4,7 +4,6 @@ exclude:
   - package.json
   - show
   - test
-  - styleguide-template
```

Comment:
This is a larger question about this `_maps.yml` file: what is it used for?

Reply:
I don't know, we should put it in the question doc.